src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,43 @@
+.menu {
+
+  background-color: $secondary-color;
+  height: 100vh;
+
+  &__icon {
+    @include top-actions;
+    margin-bottom: 40px;
+
+    @include Tablet {
+      margin-bottom: 100px;
+    }
+  }
+
+  &__list {
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
+    font-weight: 600;
```

Comment:
Remove the 'text-decoration' property as it is not needed for an unordered list and does not have any effect.